pkg/dao/repository_configs.go

```diff
@@ -393,10 +395,13 @@ func (r repositoryConfigDaoImpl) ValidateParameters(orgId string, params api.Rep
 		err = r.validateUrl(orgId, url, &response, excludedUUIDS)
 		if err != nil {
 			return response, err
-		} else if response.URL.Valid {
-			repomd := r.validateMetadataPresence(url, &response)
+		}
+		if response.URL.Valid {
+			r.yumRepo.Clear()
```

Comment:
This method gets called via a go routine in the handler, which likely means if you pass in 4 repositories to validate, it uses the same instance of the dao and the same instance of yumRepo?   If i'm right, i wonder if its causing any problems?

Reply:
I hadn't considered that. I tried validating with multiple repos with different URLs. It seems to work but I'm kinda surprised that it does.